packages/material-ui-utils/src/index.ts

```diff
@@ -32,3 +32,4 @@ export {
 export { default as usePreviousProps } from './usePreviousProps';
 export { default as visuallyHidden } from './visuallyHidden';
 export { default as integerPropType } from './integerPropType';
+export { default as unstable_useResizeObserver } from './useResizeObserver';
```

Comment:
Is the "utils" package be the best host for it? I think that "utils" are for any module that can't go into "unstyled", e.g. that is used in the "system".